`wwctl configure --all` fails due to file copy error in Makefile

# Problem
`wwctl configure --all` failed with errors below due to Makefile doesn't properly copy *.efi files to `/usr/share/warewulf/ipxe` directory


```
$ wwctl configure --all

<OMIT LINES>
Writing PXE files to: /var/lib/tftpboot/warewulf
[ERROR]   : Could not open source file /usr/share/warewulf/ipxe/x86.efi: open /usr/share/warewulf/ipxe/x86.efi: not a directory
[ERROR]   : open /usr/share/warewulf/ipxe/x86.efi: not a directory
[ERROR]   : open /usr/share/warewulf/ipxe/x86.efi: not a directory
```

and

```
diff staticfiles/x86.efi /usr/share/warewulf/ipxe
```

This means `/usr/share/warewulf/ipxe` isn't a directory but a flie equl to `staticfiles/x86.efi`.


# How to reproduce
I tested this on `main` tree and `v4.2.0` on Rocky Linux 8.4.
I installed Warewulf by following procedures.

```
git clone https://github.com/hpcng/warewulf.git
cd warewulf
make genconfig PREFIX=/usr BINDIR=/bin SYSCONFDIR=/etc SHAREDSTATEDIR=/var DATADIR=/usr/share
make all
make install
cat << 'EOS' > /etc/warewulf/warewulf.conf
ipaddr: 10.0.0.254
netmask: 255.255.252.0
warewulf:
  port: 9873
  secure: true
  autobuild overlays: true
  update interval: 60
  syslog: false
dhcp:
  enabled: true
  range start: 10.0.1.1
  range end: 10.0.1.255
  template: default
  systemd name: dhcpd
tftp:
  enabled: true
  tftproot: /var/lib/tftpboot
  systemd name: tftp
nfs:
  systemd name: nfs-server
  exports:
  - /home
  - /var/warewulf
EOS
```

After then

```
$ wwctl configure --all

Building overlay for warewulf-server: host
Enabling and restarting the DHCP services
Created symlink /etc/systemd/system/multi-user.target.wants/dhcpd.service → /usr/lib/systemd/system/dhcpd.service.
Building overlay for warewulf-server: host
Enabling and restarting the NFS services
Created symlink /etc/systemd/system/multi-user.target.wants/nfs-server.service → /usr/lib/systemd/system/nfs-server.service.
Updating system keys
Setting up key: ssh_host_rsa_key
Setting up key: ssh_host_dsa_key
Setting up key: ssh_host_ecdsa_key
Setting up key: ssh_host_ed25519_key
Setting up: /root/.ssh/authorized_keys
Writing PXE files to: /var/lib/tftpboot/warewulf
[ERROR]   : Could not open source file /usr/share/warewulf/ipxe/x86.efi: open /usr/share/warewulf/ipxe/x86.efi: not a directory
[ERROR]   : open /usr/share/warewulf/ipxe/x86.efi: not a directory
[ERROR]   : open /usr/share/warewulf/ipxe/x86.efi: not a directory
```

# How to resolve

Apply this patch

```
diff --git a/Makefile b/Makefile
index 7bf11b7..a5a9835 100644
--- a/Makefile
+++ b/Makefile
@@ -168,7 +168,7 @@ files: all
 	install -d -m 0755 $(DESTDIR)$(WWDOCDIR)
 	install -d -m 0755 $(DESTDIR)$(FIREWALLDDIR)
 	install -d -m 0755 $(DESTDIR)$(SYSTEMDDIR)
-	install -d -m 0755 $(DESTDIR)$(WWDATADIR)
+	install -d -m 0755 $(DESTDIR)$(WWDATADIR)/ipxe
 	test -f $(DESTDIR)$(WWCONFIGDIR)/warewulf.conf || install -m 644 etc/warewulf.conf $(DESTDIR)$(WWCONFIGDIR)
 	test -f $(DESTDIR)$(WWCONFIGDIR)/nodes.conf || install -m 644 etc/nodes.conf $(DESTDIR)$(WWCONFIGDIR)
 	cp -r etc/examples $(DESTDIR)$(WWCONFIGDIR)/
@@ -185,10 +185,10 @@ files: all
 	install -m 0644 LICENSE.md $(DESTDIR)$(WWDOCDIR)
 	cp bash_completion.d/warewulf $(DESTDIR)$(BASHCOMPDIR)
 	cp man_pages/* $(DESTDIR)$(MANDIR)/man1/
-	install -c -m 0644 staticfiles/arm64.efi $(DESTDIR)$(WWDATADIR)/ipxe
-	install -c -m 0644 staticfiles/i386.efi $(DESTDIR)$(WWDATADIR)/ipxe
-	install -c -m 0644 staticfiles/i386.kpxe $(DESTDIR)$(WWDATADIR)/ipxe
-	install -c -m 0644 staticfiles/x86.efi $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/arm64.efi $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/i386.efi $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/i386.kpxe $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/x86.efi $(DESTDIR)$(WWDATADIR)/ipxe

 init:
 	systemctl daemon-reload

```

--- a/Makefile
+++ b/Makefile
@@ -168,7 +168,7 @@ files: all
 	install -d -m 0755 $(DESTDIR)$(WWDOCDIR)
 	install -d -m 0755 $(DESTDIR)$(FIREWALLDDIR)
 	install -d -m 0755 $(DESTDIR)$(SYSTEMDDIR)
-	install -d -m 0755 $(DESTDIR)$(WWDATADIR)
+	install -d -m 0755 $(DESTDIR)$(WWDATADIR)/ipxe
 	test -f $(DESTDIR)$(WWCONFIGDIR)/warewulf.conf || install -m 644 etc/warewulf.conf $(DESTDIR)$(WWCONFIGDIR)
 	test -f $(DESTDIR)$(WWCONFIGDIR)/nodes.conf || install -m 644 etc/nodes.conf $(DESTDIR)$(WWCONFIGDIR)
 	cp -r etc/examples $(DESTDIR)$(WWCONFIGDIR)/
@@ -185,10 +185,10 @@ files: all
 	install -m 0644 LICENSE.md $(DESTDIR)$(WWDOCDIR)
 	cp bash_completion.d/warewulf $(DESTDIR)$(BASHCOMPDIR)
 	cp man_pages/* $(DESTDIR)$(MANDIR)/man1/
-	install -c -m 0644 staticfiles/arm64.efi $(DESTDIR)$(WWDATADIR)/ipxe
-	install -c -m 0644 staticfiles/i386.efi $(DESTDIR)$(WWDATADIR)/ipxe
-	install -c -m 0644 staticfiles/i386.kpxe $(DESTDIR)$(WWDATADIR)/ipxe
-	install -c -m 0644 staticfiles/x86.efi $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/arm64.efi $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/i386.efi $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/i386.kpxe $(DESTDIR)$(WWDATADIR)/ipxe
+	install -m 0644 staticfiles/x86.efi $(DESTDIR)$(WWDATADIR)/ipxe
 
 init:
 	systemctl daemon-reload

